lib/modules/datasource/bitrise/readme.md

```diff
@@ -0,0 +1,22 @@
+This datasource allows to fetch Bitrise steps from Git repositories.
+
+**Currently only Github is support**
+
+As `packageName` the step name expected e.g. for the following snippet the `packageName` would be `script`.
+
+`registryUrl` expects a GitHub HTTP Git Url as used below by Bitrise. See the `default_step_lib_source` field for an example.
```

Comment:
```suggestion
| Renovate field | What value to use?                      |
| -------------- | --------------------------------------- |
| `packageName`  | Name of the Bitrise step                |
| `registryUrl`  | GitHub HTTP Git URL, as used by Bitrise |

For example, in the YAML snippet below:

- `packageName` is `script`
- `registryUrl` is `https://github.com/bitrise-io/bitrise-steplib.git`
```

Let's use a table for the explanation, and a list for the examples. Please double check that the info is technically correct!